library/gmail.py

```diff
@@ -169,3 +182,130 @@ def __get_attachments(self, message: dict, userId='me'):
 
             if len(attachments) > 0:
                 message['attachments'] = attachments
+
+
+    def get_document_ids(self, type):
+        with self.service(GoogleSchemas.DRIVE) as service:
+            results = (
+                service.files()
+                .list(q = self.MIME_TYPE[type],
+                    corpora='user',
+                    includeItemsFromAllDrives=True,
+                    supportsAllDrives=True,
+                    pageSize=1)
```

Comment:
Currently I've set the pageSize parameter to 1 which will only get 1 document of the mimetype specified but this can be changed to any number.

Reply:
I think we should be getting all of them, honestly. This API work is less about having a production system than getting the data we need to inform our agent into our system

Reply:
I think that's fair. If I remove the pageSize argument, it pulls in all of them. I'll make that change!